<launch>
    <!-- Set to your sensor's range -->

    <node pkg="frontier_exploration" type="explore_client" name="explore_client" output="screen"/>

    <node pkg="frontier_exploration" type="explore_server" name="explore_server" output="screen" >

        <param name="frequency" type="double" value="2.0"/>

        <param name="goal_aliasing" type="double" value="1.0"/>

        <rosparam file="$(find fembots)/config/exploration_costmap.yaml" command="load" />
    </node>
</launch>
